components/oval-button/OvalButton.style.js

```diff
@@ -14,6 +14,11 @@ const Button = styled.a.attrs({
   width: clamp(7.5em, 10vw, 8em);
   height: 3.8em;
 
+  transition: background-color 0.3s linear;
+  -moz-transition: background-color 0.3s linear;
```

Comment:
it seems transition is now supported for all browsers except opera so we can just use `transition` (I'd double check though)